libyui-ncurses/Makefile.repo

```diff
@@ -20,7 +20,7 @@ build-hint:
 configure:
 	mkdir build; \
 	cd build; \
-	cmake ..
+	cmake -DCMAKE_INSTALL_PREFIX=/usr ..
```

Comment:
That means that you ALWAYS have to type this stuff when you are not using Makefile.repo.
This is a bad idea. Not only do you have to type it all the time (and, worse, in all your subprojects that you are working on), you also have to memorize it.

There must be a better way to to this; one with `/usr` as a default.

Reply:
Let me try something.

Reply:
Honestly that is the standard way i've been using for ages, since old configre generated makefiles... every time you need to do something different from the default way you need to change some parameters, at configure (cmake) time. I'm sorry but i can't get  why it's bad... of course simplifying that for all the projects could be good, but leaving the developer the easy way to change (I use $HOME/TEST, sometimes $HOME/local almost never /usr/local or anything on the system)

To me look easy to have system libyui version to fix bug on stable, and a local one (or more) to work on future ones...